config/routes.rb

```diff
@@ -212,9 +212,9 @@
       # temporary keeping both routes while transitioning (renaming) display_ads => billboards
       get "/display_ads/:placement_area", to: "billboards#show"
     end
-    get "/billboards/:placement_area", to: "billboards#show"
+    get "/billboards/:placement_area", to: "billboards#show", as: :billboard
     # temporary keeping both routes while transitioning (renaming) display_ads => billboards
-    get "/display_ads/:placement_area", to: "billboards#show"
+    get "/display_ads/:placement_area", to: "billboards#show", as: :display_ad
```

Comment:
I'm wondering if we need the "as" here for the 'display_ad_route' since we don't have a display_ad_path in the app?

Reply:
that’s a good point, no need to add two new path methods when only making use of one. will edit in a minute